packages/vite/src/node/plugins/css.ts

```diff
@@ -438,6 +442,14 @@ export function cssPlugin(config: ResolvedConfig): Plugin {
       }
     },
   }
+
+  // for backward compat, make `plugin.transform` a function
+  // but still keep the `handler` property
+  // so that we can use `filter` property in the future
+  plugin.transform = transformHook.handler
+  ;(plugin.transform as any).handler = transformHook.handler
```

Comment:
For example, to handle this line:
https://github.com/unocss/unocss/blob/94efa2edc9897dcfb5f40059c11ba22ec9ca0631/packages-integrations/vite/src/modes/global/build.ts#L45

Reply:
We might need more back-compat for other transforms too

- `vite:json` https://github.com/intlify/bundle-tools/blob/d3724e19cae256219f57ac7bc364ff2e07d05502/packages/unplugin-vue-i18n/src/core/resource.ts#L196-L197
- `vite:css-post` https://github.com/hi-ogawa/unocss/blob/261d335eb3a7ebf1700f2418361a2308e240fb22/packages-integrations/vite/src/modes/chunk-build.ts#L14

(I actually hit `vite:css-post` one in my CI https://github.com/hi-ogawa/vite-environment-examples/pull/157/, but I just fixed on my side and forgot to report.)